tests/rt.conf

```diff
@@ -190,5 +190,5 @@ RUN     | datm_bulk_gefs
 RUN     | datm_mx025_cfsr                                                                                                         | - wcoss_cray jet.intel gaea.intel       | fv3 |
 RUN     | datm_mx025_gefs                                                                                                         | - wcoss_cray jet.intel                  | fv3 |
 
-COMPILE | DATM=Y S2S=Y DEBUG=Y                                                                                                    | - wcoss_cray jet.intel                  | fv3 |
+COMPILE | APP=DATM_NEMS
```

Comment:
Add | - wcoss_cray jet.intel                  | fv3 |

Reply:
addressed in 277cefb